docs/getting-started/install-resoto/kubernetes.md

```diff
@@ -51,19 +51,19 @@ Wait until the ArangoDB deployment is ready. You can check the conditions in the
 $ kubectl wait --for=condition=ready arangodeployment/single-server
 ```
 
-### Create Helm Values File
+### Configure Helm Values File
 
 ```yaml title="resoto-values.yaml"
 resotocore:
   graphdb:
     server: http://single-server:8529
 ```
 
-This is the minimum configuration, which points to an empty ArangoDB database with default username and password.
+By default in the values.yaml file, this is the minimum configuration, which points to an empty ArangoDB database with default username and password.
```

Comment:
```suggestion
This is the minimum configuration, which points to an empty ArangoDB database with default username and password.
```

Reply:
By default if you see the values.yaml, we point the server to the same URL https://github.com/someengineering/helm-charts/blob/1e6b3c3661923b59f3b86793fd906c74a5d254f4/charts/resoto/values.yaml#L89, so I dont know why are we wanting the user to make the changes which are already defaulted to the same.

Reply:
This sentence and the above can be replaced with instructions to fetch the values file from the repository.